src/uninstall.ts

```diff
@@ -0,0 +1,45 @@
+import { existsSync } from 'node:fs'
+import process from 'node:process'
+import { resolve } from 'node:path'
+import { x } from 'tinyexec'
+import { detectPackageManager } from './detect'
+
+export interface UninstallPackageOptions {
+  cwd?: string
+  dev?: boolean
+  silent?: boolean
+  packageManager?: string
+  additionalArgs?: string[]
+}
+
+export async function uninstallPackage(names: string | string[], options: UninstallPackageOptions = {}) {
```

Comment:
Seems like you didn't re-export this function?

Reply:
Ohh I missed that! Done in [`639a6ba` (#17)](https://github.com/antfu-collective/install-pkg/pull/17/commits/639a6ba34b8d671a780ce0ab2dd576fc0e62e145).